yarn.lock

```diff
@@ -7452,6 +7453,16 @@ import-in-the-middle@1.7.1:
     cjs-module-lexer "^1.2.2"
     module-details-from-path "^1.0.3"
 
+import-in-the-middle@^1.11.0:
```

Comment:
if you want to run `npx yarn-deduplicate && yarn install` then this guy will go away